### DISCUSSION TITLE
Project Key support multiple projects

### DISCUSSION THREAD

**Discussion by bduncan-work** [nbrown02/FlowViz-Jira#22] [Ideas]
Could the project key parameter be enhanced from `projectKey = 'ZZZ'` to support `projectKey in ('ZZZ', 'YYY', 'WWW')` so that the dashboard can be used to show filtered projects (one of many) or multiple projects (compilation of many)?

**Comment by nbrown02** [comment]
Thanks for this - great suggestion! This seems simple to do, I just need to see if there's a simple way to make sure the parameters work for single and/or multiple that is user friendly. With the ADO version of FlowViz I have separate template files depending on your need (single vs multiple) so have considered this before on other tools.

I'll see if I can put something together in the next week for release :)

**Comment by nbrown02** [comment]
Hi @bduncan-work - I have added a new template in the main repo - please let me know your feedback :)

**Comment by nbrown02** [comment]
This was deprecated in February 2025 to improve the accuracy of the cycle time calculation.
